tests/test_util.py

```diff
@@ -31,3 +31,60 @@ def test_to_list():
     assert util.to_list("") == []
     assert util.to_list(["a"]) == ["a"]
     assert util.to_list("a") == ["a"]
+
```

Comment:
None of these tests have any milliseconds or microseconds.
For floats, just add a decimal, like `1599653541.123` or `1599653541.123456`

Reply:
The last two tests here covers float, added more decimal values.